270_asyncio_import_main_practice/asyncio_numbers.py

```diff
@@ -0,0 +1,49 @@
+# In order to create the console for the program, an asyncio based
+# reader using json and socket will be needed.
+
+import time
+import random
+import asyncio 
+import called_script
+
+
+# This function prints a number, then waits that many seconds, then
+# Prints the number again.
+# Due to asyncio, the shorter numbers will print twice first even if called
+# second.
+async def randonNumberWait():
```

Comment:
```suggestion
async def random_num_wait():
```
Avoid mixedCase (also typo `randon` -> `random`).